scripts/kotlin-lint-check.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking code formatting"
+echo "********************************"
+
+bash ../oppia-android-tools/ktlint --android app/src/**/*.kt data/src/**/*.kt domain/src/**/*.kt testing/src/**/*.kt utility/src/**/*.kt
```

Comment:
Actually, it's a bit weird to be passing ktlint to bash like this. We should be following the ktlint team's instructions here: 

https://github.com/pinterest/ktlint#usage.

The universal command would be:

```sh
java -jar ../oppia-android-tools/ktlint --android app/src/**/*.kt data/src/**/*.kt domain/src/**/*.kt testing/src/**/*.kt utility/src/**/*.kt
```

Reply:
Done

Reply:
Resolving on behalf of ben